libr/core/cio.c

```diff
@@ -485,9 +484,8 @@ R_API int r_core_shift_block(RCore *core, ut64 addr, ut64 b_size, st64 dist) {
 	if ((st64)b_size < 1) {
 		return false;
 	}
-	shift_buf = calloc (b_size, 1);
+	shift_buf = r_calloc_logerr (b_size, 1);
```

Comment:
imho is better to have a single logerr call to chk for null and do whatever its defined at configure time instead of having a logerr wrapper for each alloc call